docs/osf-trivia-list/capsule-update.md

```diff
@@ -0,0 +1,42 @@
+# Frequenty Asked Questions about Capsule Update
+
+## Q: Why was the traditional flashrom-based method replaced by capsule-based one?
+
+A: There are many advantages of capsule-based update.
+Fist of all, it's much easier[1]:
+
+>The steps to update efi firmware are: </br>
+>1.) cat firmware.cap > /dev/efi_capsule_loader </br>
+>2.) reboot
+
+Not only is it simpler but also more efficient, because there is no need for
+manual reset and disabling all of the protections before update, thus the whole
+process takes less time. Speaking of protection the last but not least reason is
+safety, the signed coalesced data can only be accessed by signatures, verified by
+edk2. Ensuring no unauthorized activity is allowed.
+
+## Q: Is fwupd still Linux only?
+
+A: Yes, fwupd is Linux only. However, this isn't the only software that
+uses capsules to perform the update, Windows also uses it (although this would
+require creating and distributing firmware as drivers through Windows Update,
+which is beyond the scope of this project). Linux can also be compiled with
+EFI_CAPSULE_LOADER [1] which exposes an interface for sending capsules to the
+firmware by a simple write to file.
+
+## Q: Where in the process are signatures verified, and which parts are signed?
```

Comment:
This doesn't contain answer to `which parts are signed` part. In fact, that part may be changed due to an issue reported by Sergii, so for the time being it would be safest to remove it.

```suggestion
## Q: Where in the process are signatures verified?
```